.travis.yml

```diff
@@ -1,27 +1,23 @@
 language: python
 
 python:
-    - "2.7"
+    - 2.7
+    - 3.4
+    - 3.5
+    - 3.6
     - pypy
-    - "3.3"
-    - "3.4"
-    - "3.5"
-    - "3.6"
     - pypy3
 
 install:
     - pip install tox-travis
 
 matrix:
+    include:
+        # Workaround to enable Python 3.7: https://github.com/travis-ci/travis-ci/issues/9815
+        - python: 3.7
+          dist: xenial
+          sudo: true
     allow_failures:
```

Comment:
do we need this??

Reply:
We used this to bypass failures on some configurations mostly due to temporary bugs. This fix removes all items currently here but I foresee we might need this in the future. It could be helpful to keep this stub to make it easier. I'll add some comment here.